client/src/components/to-do-list/Tasks.jsx

```diff
@@ -3,7 +3,7 @@ import Task from './Task.jsx';
 
 function Tasks({tasks, openModal, isMobile, deleteTask, draggedEvent, setDraggedEvent, handleDragStart}) {
   // console.log('tasks in tasks', tasks)
-  const filteredTasks = tasks.filter(task => !task.in_calendar)
```

Comment:
Ok, that is a good name change (last one I noticed was better too--easier to know what you're looking at).